### PR TITLE
Notification/local notify

### PR DESCRIPTION
**Pull request by divyansh0908**
Date: 29-09-2024

Developer Name: @divyansh0908

---

## Issue Ticket Number

https://github.com/Real-Dev-Squad/mobile-app/issues/438

## Description

This update marks the completion of the remaining tasks from [TCR](https://status.realdevsquad.com/tasks/ZRyXdtDmnEauzXOUn5eF). The rest of the changes made for this TCR can be found in this (https://github.com/Real-Dev-Squad/mobile-app/pull/439)[PR]. Additionally, This update also include a script that creates `google-services.json`, with support from env.

### Documentation Updated?

- [ ] Yes
- [ ] No

### Under Feature Flag

- [x] Yes
- [ ] No

### Database Changes

- [ ] Yes
- [x] No

### Breaking Changes

- [ ] Yes
- [x] No

### Development Tested?

- [x] Yes
- [ ] No

## Screenshots
<details>
<summary>Screenshot 1</summary>

</details>

Screenshot 2024-10-02 at 9 32 33 AM
Screenshot 2024-10-02 at 9 52 10 AM

https://github.com/user-attachments/assets/38e362a2-a67d-49fe-b372-3e303ad79220

## Test Coverage
<details>
<summary>Screenshot 1</summary>

</details>

Screenshot 2024-10-01 at 10 46 40 PM
Screenshot 2024-10-01 at 10 54 15 PM

### CODE REVIEW

File: src/components/Notify/NotifyForm.tsx [hunk 1]
```diff
@@ -43,21 +42,27 @@ const NotifyForm = ({ notifyHandler }: { notifyHandler: () => void }) => {
   };
 
   const getFCMToken = async () => {
-    const fcmToken_ = await firebase.messaging().getToken();
-    console.log('🚀 ~ getFCMToken ~ fcmToken_:', fcmToken_);
+    const permission = await firebase.messaging().hasPermission();
+    console.log('nfdeobns', permission);
+    if (permission) {
+      const fcmToken_ = await firebase.messaging().getToken();
+      console.log('🚀 ~ getFCMToken ~ fcmToken_:', fcmToken_);
```
Comment (ashifkhn) [inline src/components/Notify/NotifyForm.tsx]: Why are we keeping console here?

Comment (ashifkhn) [reply src/components/Notify/NotifyForm.tsx]: Please remove console.

File: src/screens/NotifyScreen/NotifyScreen.tsx [hunk 2]
```diff
@@ -32,4 +18,16 @@ const styles = StyleSheet.create({
   container: {
     padding: 16,
   },
+  title: {
```
Comment (ashifkhn) [inline src/screens/NotifyScreen/NotifyScreen.tsx:22]: There is the scale function for having responsive font, please use that.

File: src/service/googleServiceTemplate.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            },
+          ],
+        },
+      },
+    },
+  ],
+  configuration_version: '1',
+};
+
+console.log('🥲🥲🥲🥲🥲', googleServicesTemplate);
```
Comment (ashifkhn) [inline src/service/googleServiceTemplate.js]: Please remove console.

**Review by ashifkhn — CHANGES_REQUESTED** [review]

**Review by ashifkhn — APPROVED** [review]

**Review by yesyash — APPROVED** [review]

**Pull request merged**